database/factories/MismatchFactory.php

```diff
@@ -48,4 +62,11 @@ private function getRandomValue()
             $this->faker->words($randomWordAmount, true)
         ]);
     }
+
+    private function getRandomReviewStatus()
+    {
+        $reviewStatuses = [ 'wikidata', 'external', 'both', 'none' ];
+        $randomStatusIndex = $this->faker->numberBetween(0, 3);
+        return $reviewStatuses[ $randomStatusIndex ];
```

Comment:
Nice! We can also use [faker's  `randomElement()`](https://fakerphp.github.io/formatters/numbers-and-strings/#randomelement) for this purpose:

```suggestion
        return $this->faker->randomElement([
            'wikidata',
            'external',
            'both',
            'none'
        ]);
```

Reply:
ah, that's better. thx.